scripts/notifyRewardAmount.js

```diff
@@ -0,0 +1,85 @@
+// ES5 style
+const config = require("./deploymentConfig");
+const thorify = require("thorify").thorify;
+const Web3 = require("web3");
+const Multirewards = require(config.pathToMultirewardsJson);
+const IERC20 = require(config.pathToIERC20Json);
+const assert = require('assert');
+
+let network = null;
+let multirewardsAddress = null;
+let rewardAmount = null;
+
+if (process.argv.length < 5) 
+{
+    console.error("Usage: node scripts/notifyRewardAmount [mainnet|testnet] [Multirewards address] [Reward amount (excluding 18 decimals)]");
+    process.exit(1);
+} 
+else
+{
+    if (process.argv[2] == "mainnet") network = config.network.mainnet;
+    else if (process.argv[2] == "testnet") network = config.network.testnet;
+    else {
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
+
+    multirewardsAddress = process.argv[3];
+    rewardAmount = Web3.utils.toWei(process.argv[4]);
+}
+
+const web3 = thorify(new Web3(), network.rpcUrl);
+web3.eth.accounts.wallet.add(config.privateKey);
+
+notifyRewardAmount = async() =>
```

Comment:
Could we just immediately notifyReward after we addReward or am I misunderstanding the purpose of those two functions?

Reply:
I thought about this too. Functionally you can immediately `notifyReward` after `addReward`. We could combine the two scripts, but we shouldn't.

Cuz when we want to extend the reward period for another 3 months by adding more VEX to the contract, we'll just call `notifyReward`, and not `addReward`. So having separate scripts makes it easier in the future